Gemfile.lock

```diff
@@ -3,49 +3,42 @@ GEM
   specs:
     CFPropertyList (3.0.6)
       rexml
-    activesupport (7.1.2)
-      base64
-      bigdecimal
+    activesupport (7.0.7.2)
```

Comment:
The gem file seems to downgrade the dependencies. Is this done by mistake to meant to be downgraded?